src/main/java/bio/terra/cli/command/helperclasses/FormatFlag.java

```diff
@@ -0,0 +1,101 @@
+package bio.terra.cli.command.helperclasses;
+
+import bio.terra.cli.command.exception.SystemException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import java.io.PrintStream;
+import java.util.function.Consumer;
+import picocli.CommandLine;
+
+/**
+ * Command helper class that defines the --format flag and provides utility methods to printing a
+ * return value out in different formats.
+ */
+public class FormatFlag {
+
+  @CommandLine.Option(
+      names = "--format",
+      hidden = true,
+      description = "Set the format for printing command output: ${COMPLETION-CANDIDATES}")
+  private FormatOptions format;
+
+  /** This enum specifies the format options for printing the command output. */
+  private enum FormatOptions {
+    json,
+    text;
+  }
+
+  // output stream to use for writing command return value
+  private static final PrintStream OUT = System.out;
```

Comment:
not going to block the review on it, but I think we want this to be the same as what's in `CommandSetup`. These could both be initialized from some `OutputUtils.OUT` or something else.

Reply:
Yes, agreed. Added a ticket to track this follow-on work. [PF-603](https://broadworkbench.atlassian.net/browse/PF-603)